tests/ui/components/results/universalresultscomponent.js

```diff
@@ -0,0 +1,33 @@
+import { mount } from 'enzyme';
+import mockManager from '../../../setup/managermocker';
+import DOM from '../../../../src/ui/dom/dom';
+import StorageKeys from '../../../../src/core/storage/storagekeys';
+import SearchStates from '../../../../src/core/storage/searchstates';
+import UniversalResultsComponent from '../../../../src/ui/components/results/universalresultscomponent';
+
+DOM.setup(document, new DOMParser());
+let COMPONENT_MANAGER, defaultConfig;
+beforeEach(() => {
+  COMPONENT_MANAGER = mockManager();
+  const bodyEl = DOM.query('body');
+  DOM.empty(bodyEl);
+  DOM.append(bodyEl, DOM.createEl('div', { id: 'test-component' }));
+
+  defaultConfig = {
+    container: '#test-component'
+  };
+});
+
+describe('UniversalResults component', () => {
+  it('listens to updates to UNIVERSAL_RESULTS in global storage', () => {
```

Comment:
Do we already have a test for the `Component` base class that verifies it re-renders when the `storage.[moduleId]` is updated? If not, I think that would be useful. It would save us needing to write these kinds of tests for every component.

Reply:
no, but that makes sense, I can add it instead of this!